docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to Decentraland's catalyst
+
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+
+## We Develop with Github
+
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow), So All Code Changes Happen Through Pull Requests
+
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
```

Comment:
I get redirected from the [Github Flow](https://guides.github.com/introduction/flow/index.html) link to another page. 
It may also be repeated from the title link